snooty.toml

```diff
@@ -1,4 +1,4 @@
-name = "docs-commandline-tools"
+name = "database-tools"
```

Comment:
• Can we do this without changing the GH  repo name? Would love to do this, but not sure if we can. You're the expert!

Reply:
This is entirely unrelated to the github repo, the `name` in snooty.toml has to match a value in the autobuilder config, which is set to database-tools.